packages/wallet/src/services/WalletUtil.ts

```diff
@@ -104,3 +109,33 @@ export const createWalletUtil = (context: WalletUtilContext) => ({
 });
 
 export type WalletUtil = ReturnType<typeof createWalletUtil>;
+
+type SetWalletUtilContext = (context: WalletUtilContext) => void;
+
+export const createLazyWalletUtil = (): WalletUtil & { initialize: SetWalletUtilContext } => {
```

Comment:
Let's add some docs here

Reply:
Done in 426da027e260bcbd9bb6608b2438b15df6698450